introduction.md

```diff
@@ -1,2 +1,7 @@
 Hey, I`m Lucas, the next ninja! 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?